app/src/main/java/de/danoeh/antennapod/fragment/swipeactions/AddToQueueSwipeAction.java

```diff
@@ -0,0 +1,47 @@
+package de.danoeh.antennapod.fragment.swipeactions;
+
+import android.content.Context;
+
+import androidx.fragment.app.Fragment;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.storage.DBWriter;
+import de.danoeh.antennapod.model.feed.FeedItem;
+import de.danoeh.antennapod.model.feed.FeedItemFilter;
+
+public class AddToQueueSwipeAction implements SwipeAction {
+
+    @Override
+    public String id() {
+        return ADD_TO_QUEUE;
+    }
+
+    @Override
+    public int actionIcon() {
+        return R.drawable.ic_playlist;
+    }
+
+    @Override
+    public int actionColor() {
+        return R.attr.colorAccent;
+    }
+
+    @Override
+    public String title(Context context) {
```

Comment:
I think it would be better to use a more explanatory method name here, such as `getTitle` instead of just `title`. Otherwise, it is not clear that the `title` method *returns* the title but the `action` method *performs* the action.

Reply:
Same for the id, icon and color methods